src/watcher.js

```diff
@@ -29,14 +30,20 @@ declare function exports(params: OnSourceChangeParams): Watchpack;
 
 export type OnSourceChangeFn = (params: OnSourceChangeParams) => Watchpack;
 
-export default function onSourceChange(
-  {sourceDir, artifactsDir, onChange, shouldWatchFile}: OnSourceChangeParams
+export default function onSourceChange({
+  sourceDir, artifactsDir, onChange, shouldWatchFile, reloadDelay,
+}: OnSourceChangeParams
 ): Watchpack {
   // TODO: For network disks, we would need to add {poll: true}.
   const watcher = new Watchpack();
 
   const executeImmediately = true;
-  onChange = debounce(onChange, 1000, executeImmediately);
+
+  if (reloadDelay) {
+    onChange = debounce(onChange, reloadDelay, executeImmediately);
```

Comment:
Please read step 4 of https://github.com/mozilla/web-ext/issues/1573#issuecomment-496253303 again. If you don't understand the difference, then https://github.com/mozilla/web-ext/issues/1137#issuecomment-364833409 might provide more context. If it's still not clear after reading that, feel free to ask for clarification here.